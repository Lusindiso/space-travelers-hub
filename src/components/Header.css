
.flex-space {
  display: flex;
  justify-content: space-between;
  padding: 1rem;
}

.left-nav {
  display: flex;
  align-items: center;
  gap: 1rem;
}

.right-nav {
  display: flex;
  align-items: center;
  gap: 1rem;
}

.right-nav a {
  color: blue;
}

.missions {
  border-right: solid black;
  padding-right: 1rem;
}

.rockets {
  border-right: solid black;
  padding-right: 1rem;
}

header {
  border-bottom: 1px solid rgb(119 114 114);
}
