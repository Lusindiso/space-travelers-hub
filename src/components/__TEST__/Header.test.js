import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import Header from '../Header';

describe('Testing navBar component', () => {});
it('should render correctly NavBar component', () => {
  const tree = render(
    <Router>
      <Header />
    </Router>,
  );
  expect(tree).toMatchSnapshot();
});
