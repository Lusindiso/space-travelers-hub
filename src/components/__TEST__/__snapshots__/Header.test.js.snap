// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly NavBar component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <div
          class="navbar-space"
        >
          <nav
            class="space-btw"
          >
            <ul
              class="flex-space"
            >
              <div
                class="left-nav"
              >
                <li>
                  <img
                    alt="logo"
                    class="logo"
                    src="planet.png"
                  />
                </li>
                <li>
                  <h1>
                    Space Travelers' Hub
                  </h1>
                </li>
              </div>
              <div
                class="right-nav"
              >
                <li>
                  <a
                    aria-current="page"
                    class="rockets active"
                    href="/"
                    style="text-decoration: underline;"
                  >
                    Rockets
                  </a>
                </li>
                <li>
                  <a
                    class="missions"
                    href="/missions"
                  >
                    Missions
                  </a>
                </li>
                <li>
                  <a
                    class="categories"
                    href="/myProfile"
                  >
                    My Profile
                  </a>
                </li>
              </div>
            </ul>
          </nav>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <div
        class="navbar-space"
      >
        <nav
          class="space-btw"
        >
          <ul
            class="flex-space"
          >
            <div
              class="left-nav"
            >
              <li>
                <img
                  alt="logo"
                  class="logo"
                  src="planet.png"
                />
              </li>
              <li>
                <h1>
                  Space Travelers' Hub
                </h1>
              </li>
            </div>
            <div
              class="right-nav"
            >
              <li>
                <a
                  aria-current="page"
                  class="rockets active"
                  href="/"
                  style="text-decoration: underline;"
                >
                  Rockets
                </a>
              </li>
              <li>
                <a
                  class="missions"
                  href="/missions"
                >
                  Missions
                </a>
              </li>
              <li>
                <a
                  class="categories"
                  href="/myProfile"
                >
                  My Profile
                </a>
              </li>
            </div>
          </ul>
        </nav>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
